ppa/archive/gale.py

```diff
@@ -12,6 +12,22 @@
 logger = logging.getLogger(__name__)
 
 
+def get_local_ocr(item_id, page_num):
+    """
+    Get local OCR page text for specified page of a single Gale volume
+    """
```

Comment:
might be helpful to add a bit more context/info here in comments, here's a suggestion:
```suggestion
    """
    # Pull local OCR text for Gale content if available. Requires a base directory to be configured and assumes that
    # files are organized following the convention of stub directories based on every third number from the
    # Gale ID and then named based on item id and page number, matching the image file names provided by Gale.
```